assets/scripts/util/fetch_blocking.js

```diff
@@ -27,6 +31,7 @@ export function newBlockingAjaxRequest (mode, request, doneFunc, cancelFunc) {
   blockingAjaxRequestInProgress = true
 
   blockingAjaxRequest = request
+  blockingAjaxRequest.credentials = 'include'
```

Comment:
We need to refactor this entire function eventually, so I'm hesitant to make changes here that we need to "remember" and unwind. It seems to me that the only request this is handling is the POST to `v1/streets` from `remix.js`. Let's we add `credentials: 'include'` to that request object directly.